component.json

```diff
@@ -105,6 +105,21 @@
           "prompt": "Please select a Salesforce Object"
         }
       }
+    },
+    "stream": {
```

Comment:
Can we get a better name than `stream`?  Perhaps `streamPlatformEvents` to distinguish from possible future trigger `streamFromPushTopic`?

Reply:
will be changed